gambit-stuck-submissions-import/index.js

```diff
@@ -0,0 +1,81 @@
+const fetch = require('node-fetch');
+const logger = require('winston');
+const underscore = require('underscore');
+const Promise = require('bluebird');
+
+const config = require('./config');
+const GambitService = require('./lib/gambit');
+
+function getCampaignsAndKeyword(campaigns) {
+  const campaignsMap = {};
+  campaigns.forEach(campaign => {
+    campaignsMap[campaign.id] = underscore.first(campaign.keywords).keyword;
+  });
+  return campaignsMap;
+}
+
+async function getStuckSubmissionsByCampaign(campaigns) {
+  let submissions;
+  const db = await getDB();
+  const aggregator = [{
+    $match: {
+      $and: [
+        { failed_at: { $exists: true } },
+        { submitted_at: { $exists: false } },
+        { campaign: { $in: Object.keys(campaigns).map(key => Number(key)) } }
+      ]
+    },
+  },
+  {
+    $group: {
+      _id: "$campaign",
+      submissions: {
+        $push: "$$ROOT"
+      }
+    }
+  }];
+
+  // query the production gambit database to gather all stuck submissions that belong to the current running campaigns (see above)
+  try {
+    submissions = await db.collection('reportback_submissions').aggregate(aggregator).toArray();
+  } catch(error) {
+    logger.error(error);
+  }
+  return submissions;
+}
+
+// Once we have identified these stuck submissions, re POST to chatbot route with:
+// - the number of the user
+// - the madeup profile id
```

Comment:
I know we're not posting yet, so nothing we need to change before merg e-- but why include a fake profile ID? Assuming this is the `profile_id` param for an incoming Mobile Commons request?

I do wonder when we get to part 2 if we possibly want to hijack that property or someplace else to indicate this was a manual import.